Dockerfile

```diff
@@ -32,8 +30,8 @@ RUN DOCKER=true pnpm prune --production
 # Get ready for production
 FROM ghcr.io/hazmi35/node:18-alpine
 
-LABEL name "chitoge"
-LABEL maintainer "Zen <zen@frutbits.org>"
+LABEL name "matsuri"
+LABEL maintainer "Anime World Indonesia <contact@animeworld.moe>"
```

Comment:
Shouldn't we use more specified email handle, like dev@animeworld.moe?
I imagine contact@animeworld.moe will be used for general inquiries like partnership requests.